src/python/pants/backend/jvm/tasks/coverage/util.py

```diff
@@ -0,0 +1,45 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import os
+
+
+def is_coverage_target(tgt):
+  return (tgt.is_java or tgt.is_scala) and not tgt.is_test and not tgt.is_synthetic
+
+
+def initialize_instrument_classpath(settings, targets, instrumentation_classpath):
```

Comment:
This seems terribly specific to CoverageSettings and/or BaseCoverage, which wouldn't seem to make it a great candidate for a `util` package. ... And in particular, since Jacoco doesn't use an instrumentation classpath, isn't this just Cobertura specific at this point?

Reply:
moved these to static methods in cobertura, and updated test accordingly